### PR TITLE
[WIP] 398 dependency manager consolidation

### PR DESCRIPTION
**Pull request by ryan-cahill**

### CODE REVIEW

File: .github/workflows/cli-rc.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: CLI - Cut Release Candidate
 
-on: push
+on:
```
Comment (mueschm) [inline .github/workflows/cli-rc.yml:3]: Why is this change needed?

**Review by mueschm — CHANGES_REQUESTED** [review]

File: .github/workflows/cli-rc.yml (hunk 1 above)
Comment (ryan-cahill) [reply .github/workflows/cli-rc.yml:3]: Part of this work is to make an update to the GitHub actions to publish CLI RCs on pull requests
